ddl/delete_range.go

```diff
@@ -94,7 +94,15 @@ func (dr *delRange) addDelRangeJob(ctx context.Context, job *model.Job) error {
 	}
 	defer dr.sessPool.put(sctx)
 
-	err = insertJobIntoDeleteRangeTable(ctx, sctx, job)
+	if job.MultiSchemaInfo != nil {
+		var added bool
+		added, err = insertJobIntoDeleteRangeTableMultiSchema(ctx, sctx, job)
```

Comment:
`added == false` implies that `err is nil`, it may be changed later.

Reply:
Updated.